From 624950f61ecf23f4fa66096ec20e939588017c72 Mon Sep 17 00:00:00 2001
From: Shile Zhang <shile.zhang@nokia-sbell.com>
Date: Fri, 17 Nov 2017 16:59:27 +0800
Subject: [PATCH 2/4] arm: fix the spacing/tabbing issue

To fix the style issue where spaces where used instead of tabs.

Signed-off-by: Shile Zhang <shile.zhang@nokia-sbell.com>
---
 arch/arm/Kconfig | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 51c8df5..8d536e0 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -221,7 +221,7 @@ config ZONE_DMA
 	bool
 
 config NEED_DMA_MAP_STATE
-       def_bool y
+	def_bool y
 
 config ARCH_SUPPORTS_UPROBES
 	def_bool y
@@ -1126,9 +1126,9 @@ config ARM_ERRATA_764369
 	  in the diagnostic control register of the SCU.
 
 config ARM_ERRATA_775420
-       bool "ARM errata: A data cache maintenance operation which aborts, might lead to deadlock"
-       depends on CPU_V7
-       help
+	bool "ARM errata: A data cache maintenance operation which aborts, might lead to deadlock"
+	depends on CPU_V7
+	help
 	 This option enables the workaround for the 775420 Cortex-A9 (r2p2,
 	 r2p6,r2p8,r2p10,r3p0) erratum. In case a date cache maintenance
 	 operation aborts with MMU exception, it might cause the processor
@@ -1689,12 +1689,12 @@ config HW_PERF_EVENTS
 	depends on ARM_PMU
 
 config SYS_SUPPORTS_HUGETLBFS
-       def_bool y
-       depends on ARM_LPAE
+	def_bool y
+	depends on ARM_LPAE
 
 config HAVE_ARCH_TRANSPARENT_HUGEPAGE
-       def_bool y
-       depends on ARM_LPAE
+	def_bool y
+	depends on ARM_LPAE
 
 config ARCH_WANT_GENERAL_HUGETLB
 	def_bool y
-- 
2.6.2

