From f9353ab54bc0962d98ae9b4e05087d306ba3b759 Mon Sep 17 00:00:00 2001
From: Shile Zhang <zhangshile@gmail.com>
Date: Tue, 9 Jan 2018 16:12:29 +0800
Subject: [PATCH 4/4] mm/page_alloc.c: fix typos in comments

Signed-off-by: Shile Zhang <zhangshile@gmail.com>
---
 mm/page_alloc.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/mm/page_alloc.c b/mm/page_alloc.c
index 76c9688..bfd5f99 100644
--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -293,7 +293,7 @@ int page_group_by_mobility_disabled __read_mostly;
 #ifdef CONFIG_DEFERRED_STRUCT_PAGE_INIT
 
 /*
- * Determine how many pages need to be initialized durig early boot
+ * Determine how many pages need to be initialized during early boot
  * (non-deferred initialization).
  * The value of first_deferred_pfn will be set later, once non-deferred pages
  * are initialized, but for now set it ULONG_MAX.
@@ -344,7 +344,7 @@ static inline bool update_defer_init(pg_data_t *pgdat,
 				unsigned long pfn, unsigned long zone_end,
 				unsigned long *nr_initialised)
 {
-	/* Always populate low zones for address-contrained allocations */
+	/* Always populate low zones for address-constrained allocations */
 	if (zone_end < pgdat_end_pfn(pgdat))
 		return true;
 	(*nr_initialised)++;
@@ -1502,7 +1502,7 @@ static unsigned long __init deferred_init_range(int nid, int zid,
 	 * performing it only once every pageblock_nr_pages.
 	 *
 	 * We do it in two loops: first we initialize struct page, than free to
-	 * buddy allocator, becuse while we are freeing pages we can access
+	 * buddy allocator, because while we are freeing pages we can access
 	 * pages that are ahead (computing buddy page in __free_one_page()).
 	 */
 	for (pfn = start_pfn; pfn < end_pfn; pfn++) {
@@ -3391,7 +3391,7 @@ __alloc_pages_may_oom(gfp_t gfp_mask, unsigned int order,
 	if (gfp_mask & __GFP_THISNODE)
 		goto out;
 
-	/* Exhausted what can be done so it's blamo time */
+	/* Exhausted what can be done so it's blame time */
 	if (out_of_memory(&oc) || WARN_ON_ONCE(gfp_mask & __GFP_NOFAIL)) {
 		*did_some_progress = 1;
 
-- 
2.6.2

